test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1

```diff
@@ -668,6 +678,40 @@ er
                 Write-Verbose $e.ToString() -Verbose
             }
             $ps.HadErrors | Should -BeFalse
-            $output.Replace("`r","").Replace(" ","*").Replace("`n","``") | Should -BeExactly $expectedTable.Replace("`r","").Replace(" ",".").Replace("`n","``")
+            $output.Replace("`r","").Replace(" ","*").Replace("`n","^") | Should -BeExactly $expectedTable.Replace("`r","").Replace(" ",".").Replace("`n","^")
+        }
+
+        It "Should render header correctly where values are wider than header: <variation>" -TestCases @(
+            @{ variation = "first column"; obj = [pscustomobject]@{abc="1234";bcd="123"}; expectedTable = @"
+
+abc  bcd
+---  ---
+1234 123
```

Comment:
Should 4 be on new next line?
```
abc bcd
--- ---
123 123
4

Reply:
That's only if there is a format file that specifies the width.  When there isn't it will auto set the width of a column based on the first few samples of the output.  So in this case, the `1234` dictate the width of column 1.

Reply:
Is the fix (and the bug) for autosize only?

Reply:
Could you please add a comment or change the test header that it is a _implicitly autosize_?
(Also we should add later tests for explicit `-AutoSize`.)

Reply:
Updated test description to make clear and added more rows to cover accidental trim of a row and not just header